src/Pages/AboutPage.jsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+
+import {
+  Container,
+  Grid,
+  Typography,
+  Box,
+  CardContent,
+  Card,
+  makeStyles,
+} from "@material-ui/core";
+import TeamCard from "../Components/TeamCard";
+import PrimaryButton from "../ui-kit/Button";
+import Footer from "../Components/Footer";
+
+const team = [1, 2, 3, 4, 5, 6];
+
+const useStyles = makeStyles({
+  root: {
+    backgroundColor: "#C4C4C4",
```

Comment:
let's reference the color variables that were created in `src/CSS/constants.css`